src/api/elected-representative.js

```diff
@@ -2,7 +2,21 @@ import { apiClient } from 'services/networking/client'
 import qs from 'qs'
 import { newPaginatedResult } from 'api/pagination'
 
-export const getElected = async filter => {
+export const getAllElected = async filter => {
   const data = await apiClient.get(`api/v3/elected_representatives?${qs.stringify(filter)}`)
   return newPaginatedResult(data.items, data.metadata)
 }
+
+export const getElected = async uuid => {
+  const data = await apiClient.get(`/api/v3/elected_representatives/${uuid}`)
+  return data
```

Comment:
la variable `data` n'est pas très utile ici 
```suggestion
  return await apiClient.get(`/api/v3/elected_representatives/${uuid}`)
```